src/utils/storage/custom/tokenStorage.ts

```diff
@@ -15,7 +15,7 @@ export enum ITokenEnum {
 
 interface ITokenStorage {
   set: (response: ITokenResponse) => void;
-  get: (tokenEnum: ITokenEnum) => string | null;
+  get: (tokenEnum: ITokenEnum) => string | null | undefined;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The return type of the `get` method in the `ITokenStorage` interface has been changed to include `undefined`. This change could potentially introduce bugs if not all parts of the code that use this method are properly updated to handle an `undefined` return value. Please ensure that all usages of this method are checked and updated accordingly.
```diff
-  get: (tokenEnum: ITokenEnum) => string | null;
+  get: (tokenEnum: ITokenEnum) => string | null | undefined;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
Closed